generated/language/src/main/java/com/google/cloud/language/v1/spring/LanguageServiceSpringAutoConfiguration.java

```diff
@@ -0,0 +1,471 @@
+    RetrySettings.Builder classifyTextRetrySettingBuilder =
+        clientSettingsBuilder.classifyTextSettings().getRetrySettings().toBuilder();
+    if (this.clientProperties.getClassifyTextInitialRetryDelay() != null) {
+      classifyTextRetrySettingBuilder.setInitialRetryDelay(
+          this.clientProperties.getClassifyTextInitialRetryDelay());
+      LOGGER.info(
+          "ClassifyTextInitialRetryDelay set to "
+              + this.clientProperties.getClassifyTextInitialRetryDelay());
+    }
+    if (this.clientProperties.getClassifyTextRetryDelayMultiplier() != null) {
+      classifyTextRetrySettingBuilder.setRetryDelayMultiplier(
+          this.clientProperties.getClassifyTextRetryDelayMultiplier());
+      LOGGER.info(
+          "ClassifyTextRetryDelayMultiplier set to "
+              + this.clientProperties.getClassifyTextRetryDelayMultiplier());
+    }
+    if (this.clientProperties.getClassifyTextMaxRetryDelay() != null) {
+      classifyTextRetrySettingBuilder.setMaxRetryDelay(
+          this.clientProperties.getClassifyTextMaxRetryDelay());
+      LOGGER.info(
+          "ClassifyTextMaxRetryDelay set to "
+              + this.clientProperties.getClassifyTextMaxRetryDelay());
+    }
+    if (this.clientProperties.getClassifyTextInitialRpcTimeout() != null) {
+      classifyTextRetrySettingBuilder.setInitialRpcTimeout(
+          this.clientProperties.getClassifyTextInitialRpcTimeout());
+      LOGGER.info(
+          "ClassifyTextInitialRpcTimeout set to "
+              + this.clientProperties.getClassifyTextInitialRpcTimeout());
+    }
+    if (this.clientProperties.getClassifyTextRpcTimeoutMultiplier() != null) {
+      classifyTextRetrySettingBuilder.setRpcTimeoutMultiplier(
+          this.clientProperties.getClassifyTextRpcTimeoutMultiplier());
+      LOGGER.info(
+          "ClassifyTextRpcTimeoutMultiplier set to "
+              + this.clientProperties.getClassifyTextRpcTimeoutMultiplier());
+    }
+    if (this.clientProperties.getClassifyTextMaxRpcTimeout() != null) {
+      classifyTextRetrySettingBuilder.setMaxRpcTimeout(
+          this.clientProperties.getClassifyTextMaxRpcTimeout());
+      LOGGER.info(
+          "ClassifyTextMaxRpcTimeout set to "
+              + this.clientProperties.getClassifyTextMaxRpcTimeout());
+    }
+    if (this.clientProperties.getClassifyTextTotalTimeout() != null) {
+      classifyTextRetrySettingBuilder.setTotalTimeout(
+          this.clientProperties.getClassifyTextTotalTimeout());
+      LOGGER.info(
+          "ClassifyTextTotalTimeout set to " + this.clientProperties.getClassifyTextTotalTimeout());
+    }
+    clientSettingsBuilder
+        .classifyTextSettings()
+        .setRetrySettings(classifyTextRetrySettingBuilder.build());
+    RetrySettings.Builder annotateTextRetrySettingBuilder =
+        clientSettingsBuilder.annotateTextSettings().getRetrySettings().toBuilder();
+    if (this.clientProperties.getAnnotateTextInitialRetryDelay() != null) {
```

Comment:
@meltsufin this is probably the type of code for which the duplication discussion came up. 
IMHO, there are two reasons people read code: one is to modify the code, and the other is to troubleshoot. For the first one one (code modification), there is an obvious benefit to avoiding code duplication ("keeping it DRY") -- if one piece of duplicated code is changed and the other is not, it will lead to bugs. However, this scenario does not apply to autogenerated code, per @blakeli0 's point.
The second scenario -- troubleshooting -- actually benefits from this type of code duplication, as the users can identify the exact set of properties that caused them issues. If duplication were avoided, a loop is needed, so users will then need a conditional breakpoint, which is a bit harder.